components/math/MultiplicationBlock.tsx

```diff
@@ -1,15 +1,30 @@
 import React, { useEffect, useState } from "react";
 
-export default function MultiplicationBlock() {
+export interface MultiplicationBlockProps {
+  text: string;
+  selected?: boolean;
```

Comment:
the question mark means that this property is optional. Are you sure that it should be optional? same question for the onclick prop below